logging/FileAndFolderHandler_unittest.cpp

```diff
@@ -0,0 +1,79 @@
+#include <gtest/gtest.h>
+#include "FileAndFolderHandler.hpp"
+
+#include <cstdio>
+#include <unistd.h>
+
+namespace logging
```

Comment:
For every tests here please create the files in `/tmp` as this would avoid any bad collisions

Reply:
how can I do that instead of hard coding the path ?